greet.js

```diff
@@ -0,0 +1,8 @@
+export function greet(name) {
```

Comment:
You could use here a `default parameters` to be sure not have an `undefined` argument.
```suggestion
export function greet(name="") {
```